src/Component/Symbolizer/Field/ImageField/ImageField.tsx

```diff
@@ -158,7 +157,15 @@ export const ImageField: React.FC<ImageFieldProps> = ({
         />
         {
           !disableSprite &&
-            <FieldSet title={spriteTooltipLabel} checked={isSprite(value)} onCheckChange={onCheckSpriteChange} >
```

Comment:
It seems the `Fieldset` component was used only here and can be removed form GeoStyler repository